sample/src/extension.ts

```diff
@@ -6,7 +6,7 @@
 import * as cp from 'child_process';
 import * as path from 'path';
 import * as vscode from 'vscode';
-import * as runtimeExtension from 'vscode-dotnet-runtime'; // comment this out when packing the extension
+// import * as runtimeExtension from 'vscode-dotnet-runtime'; // comment this out when packing the extension
```

Comment:
silly question, but doe TS support any preprocessor directives like `#ifdef` to wrap the import around and pass a value through to the compiler to make it true for packing, but undefined for building?

Reply:
That would be pretty cool... it looks like it does not but there are tools out there that add a build step to mimic that behavior.